src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -5957,6 +5958,8 @@ Note* MusicXMLParserPass2::note(const String& partId,
             auto noteheadValue = m_e.readText();
             if (noteheadValue == "none") {
                 hasHead = false;
+            } else if (noteheadValue == "named") {
```

Comment:
Such a notehead type doesn't exist in MusicXML, see https://www.w3.org/2021/06/musicxml40/musicxml-reference/elements/notehead/ and https://www.w3.org/2021/06/musicxml40/musicxml-reference/data-types/notehead-value/
Esp. this:
Noteheads in the SMuFL [Note name noteheads](https://www.w3.org/2021/03/smufl14/tables/note-name-noteheads.html) and [Note name noteheads supplement](https://www.w3.org/2021/03/smufl14/tables/note-name-noteheads-supplement.html) ranges (U+E150–U+E1AF and U+EEE0–U+EEFF) should not use the smufl attribute or the other value, but instead use the <notehead-text> element.

Reply:
Apologies, this is an out of spec case for a single exporter

Reply:
Ah, I see, thanks